lxd/storage/drivers/volume.go

```diff
@@ -418,6 +421,14 @@ func (v Volume) NewVMBlockFilesystemVolume() Volume {
 		newConf["size"] = v.driver.Info().DefaultVMBlockFilesystemSize
 	}
 
+	// Generate a new UUID by adding 1 to the last byte of the UUID.
```

Comment:
We dont need this anymore right?

Reply:
It is essential in order to create a repeatable UUID for the VM's filesystem volume. The root volumes UUID is already occupied. By adding +1 to its UUID we get another one which is dedicated for the filesystem volume.

Reply:
But we already discussed that the config filesystem volume for a VM would have a different extension that the block volume, so this isnt needed anymore right?

Reply:
I see what you are saying. This makes having another UUID obsolete. Let me change this, thanks!

Reply:
This is now removed as discussed.